containers/ecr-viewer/src/app/data/db/schemas/common/19700101000000_initial.ts

```diff
@@ -0,0 +1,86 @@
+// import { Kysely } from "kysely";
+import { Kysely } from "kysely";
+
+import { dbNamespace } from "@/app/api/services/database";
+import { getSql } from "@/app/api/services/dialects/common";
+
+const schema = dbNamespace();
```

Comment:
I think we probably need to move this off of the top level to make sure it's getting the current environment variables given this gets set up via instrumentation - have you run into any issues with that?